iati-activities-schema.xsd

```diff
@@ -270,25 +264,28 @@
       <xsd:attribute name="type" use="optional" type="xsd:string">
         <xsd:annotation>
           <xsd:documentation xml:lang="en">
-            The type of organisation issuing the report. See IATI
-            codelist for values.
+            The type of organisation issuing the report. See IATI codelist for values.
           </xsd:documentation>
         </xsd:annotation>
       </xsd:attribute>
       <xsd:attribute name="role" type="xsd:string" use="required">
         <xsd:annotation>
           <xsd:documentation xml:lang="en">
-            An IATI code describing the organisation's role in the
-            activity (donor, agency, etc.).
+            An IATI code describing the organisation's role in the activity (donor, agency, etc.).
           </xsd:documentation>
         </xsd:annotation>
       </xsd:attribute>
       <xsd:attribute name="activity-id" type="xsd:string" use="optional">
         <xsd:annotation>
           <xsd:documentation xml:lang="en">
-            A valid activity identifier published by the participating
-            organisation which points to the activity that it has
-            published to IATI that describes its role in this activity.
+            A valid activity identifier published by the participating organisation which points to the activity that it has published to IATI that describes its role in this activity.
+          </xsd:documentation>
+        </xsd:annotation>
+      </xsd:attribute>
+      <xsd:attribute name="crs-channel-code" type="xsd:string" use="optional">
+        <xsd:annotation>
+          <xsd:documentation xml:lang="en">
+            Under CRS++ Reporting Directives this code identifies the implementing agency. Codes ending in ‘00’ are generic and are similar to the OrganisationType code.
```

Comment:
Incorrect type of punctuation around `00`.

Reply:
Looking to find an equivalent example to determine which type of tick or quote should be used instead.

Reply:
It should instead use `'`.

Reply:
This is taken from the description for http://iatistandard.org/202/activity-standard/iati-activities/iati-activity/related-activity/ which also quotes something in a similar manner.